src/megachatapi_impl.cpp

```diff
@@ -8059,8 +8112,16 @@ bool MegaChatCallHandler::isParticipating(Id userid)
 
 void MegaChatCallHandler::removeAllParticipants()
 {
-    chatCall->removeAllParticipants();
-    megaChatApi->fireOnChatCallUpdate(chatCall);
+    MegaHandleList* clientids = chatCall->getClientidParticipants();
```

Comment:
Alright, let's make clear for apps devs that, upon a reconnecting status, these callbacks can and should be ignored (the participants have not left/drop the call, but you did).
Additionally, check that the addition of the existing participants to the call during the transition from _Reconnecting_ to _In progress_ happens before the new status _In progress_ is notified (so the apps also ignore these callbacks until the call is fully established again). It should happen like that already, since upon reconnection the calldata of each current participant is received before the own user starts to re-join/reconnect the call.

Reply:
Added explanation to documentation